NavigationReact/sample/relay-modern/README.md

```diff
@@ -0,0 +1,11 @@
+# Navigation Relay (Modern)
```

Comment:
I think we should change the name to "Navigation Relay Modern". Do you think we need the brackets?